internal/donor/donordata/provided.go

```diff
@@ -193,6 +193,10 @@ type Provided struct {
 	// VoucherInvitedAt records when the invite is sent to the voucher to vouch.
 	VoucherInvitedAt time.Time `checkhash:"-"`
 
+	// DetailsVerifiedByVoucher records that a voucher has verified details supplied by
+	// the donor match their identity.
+	DetailsVerifiedByVoucher bool `checkhash:"-"`
```

Comment:
This was the least bad option - the alternative was to have to get and pass the voucher into every page that's using `CanChangePersonalDetails()`